### PR TITLE
Tidy

### PR DESCRIPTION
**Pull request by sjg20**
Tidy up / improve (I believe) a few things I noticed on first reading

### CODE REVIEW

**Review by jmarinho — APPROVED** [review]
Minor comment, otherwise looks good to me.

File: source/transfer_list.rst [hunk 1]
```diff
@@ -7,8 +7,9 @@ Transfer list
 =============
 
 The TL is composed of a TL header which is followed by sequence of Transfer
-Entries (TE). The whole TL is PA-contiguous, the TL header and all the TEs are
-16-byte aligned. The TL header specifies the number of bytes occupied by the
+Entries (TE). The whole TL is contiguous in physical address space. The TL
+header and all the TEs are 16-byte aligned (we use align16() to denote this).
```
[the remark this answers is not in the document]
Comment (sjg20) [reply source/transfer_list.rst:11]: good idea, done

**Review by jmarinho — APPROVED** [review]

**Review by apalos — APPROVED** [review]

**Comment by sjg20** [comment]
Hmmm this is set up for the 'merge' approach. Is it possible to use rebase instead? I don't think the merges add much. In this case it used the wrong email address (just for the merge) too

**Comment by danh-arm** [comment]
> Is it possible to use rebase instead? 
OK. For projects with lots of contributions and a risk of subtle integration problems, using merge commits can help track down the problem. But for this project, I agree the rebase approach should be fine. I'll change it.

**Pull request merged**